### DISCUSSION TITLE
能否支持QA类型的文档

### DISCUSSION THREAD

**Discussion by panpeng840715** [chatchat-space/Langchain-Chatchat#2646] [General]
能否支持QA类型的文档

**Comment by zRzRzRzRzRzRzR** [comment]
现在就是支持的吧，现在QAcsv就能直接读取

**Reply by panpeng840715** [reply]
请问有没有接口文档说明？需要用接口向csv文件中添加数据

**Reply by panpeng840715** [reply]
![932dffa65506e69507633b5ad7cd0b9](https://github.com/chatchat-space/Langchain-Chatchat/assets/59992252/a87d1d7c-638e-426b-b8ed-2b0dc34b794c) 还有这个报错是什么引起的吗？感谢

**Reply by black-pwq** [reply]
据我目前的使用，csv文件的结构可以大致如下：
问题,答案
问题1,答案1
问题2,答案2

第一行的各个字段暗示对应的各个列的名称
第二行往下就是真正的问答对了
